tests/tests.rs

```diff
@@ -197,8 +198,8 @@ fn compare_generated_header(
         }
     }
 
-    // Override the diff.
-    {
+    // Overwrite the expectation with actual output.
+    if overwrite_expectation {
```

Comment:
Maybe this should check the env var at runtime? That means that we wouldn't need to recompile all the tests just to update the expectations.